### PR TITLE
Add gps prometheus metrics

### PR DESCRIPTION
**Pull request by abalmos**
Adds Prometheus GPS fix metrics to gps2tsdb

### CODE REVIEW

**Comment by abalmos** [comment]
This is running on dev5, dev6, and 4. Appears to work. Will be on 2 on next upgrade.

Note: Need to fix dev5, dev,6 4, and 2 to `isoblue/gps2tsdb:latest` after merge.

**Review by aaron97neu — APPROVED** [review]
LGTM and we have seen the data appear on the grafana dashboard. Merge whenever ready

**Comment by aaron97neu** [comment]
Will this close #65?

File: services/gps2tsdb/gps.py
Comment (abalmos) [inline services/gps2tsdb/gps.py:26]: @aaron97neu Is this `global` business the Python way? I didn't see a clear why to do it otherwise.

**Comment by abalmos** [comment]
> Will this close #65?

No, but #65 is essentially done. Committing the new container maintainer compose files to `oats-fleet` would be the final task for that.

File: services/gps2tsdb/gps.py
Comment (aaron97neu) [reply services/gps2tsdb/gps.py:26]: I was actually thinking about this... [PEP8](https://www.python.org/dev/peps/pep-0008/) does not mention anything about using globals. I am far from an expert on things being "pythonic" but I think they should be avoided similar to other languages. However, I am not sure how to pass that information to `fix()` unless `bus.add_signal_receiver()` allows us to specify additional parameters...

One alternative way would be to specify a single 'settings' global dictionary (Or 1 for db stuff and 1 for prometheus). The information would be accessed via `prometheus_config['lat_gauge']`, `prometheus_config['lng_gauge']`, etc. This would avoid a bunch of globals floating around, make adding additional prometheus data points easier (altitude, accuracy, etc), and if we do find a way to pass additional parameters to `fix`, we would only need one additional parameter

At the end of the day the script is readable (but could use some comments) and works. In the future I expect we will rewrite this with the Jeepney lib and it may not be a problem then. I vote we leave it

Comment (abalmos) [reply services/gps2tsdb/gps.py:26]: Fair points. Seems like a tracing lib like prometheus would have to rely on globals or just about everything would have to accept it as a parameter. 

Feels less weird to pass the `db` instance around because only certain functions need to use the DB -- but like you said, that does not appear to be possible with this DBUS setup.

In the re-write, reducing to a one or two global maps would at least be an improvement.

**Pull request merged**
